docs/artefacts/offramps.md

```diff
@@ -872,6 +872,33 @@ offramp:
 ```
 
 
+#### create_subscription
+
+Create a subsrciption to a pub/sub topic.
+
+**Request**:
+```js
+{
+  "command": "create_subscription",
+  "project": "<project-id>",
+  "topic": "<topic-name>",
+  "subscription": "<subscription-name>",
+  "message_ordering": `<message-ordering>`
+}
+```
+
+**Response**:
+```js
```

Comment:
whoopsie, does pub/sub provide an empty response? or is the response payload missing?

Reply:
It does actually :'D I had left it as a TODO. As of now, it throws an error if the subscription doesn't get created and no response if it does. I'll fix this to give a response ^_^

Reply:
Modified the docs!